using_images/db_images/mongodb.adoc

```diff
@@ -134,3 +134,91 @@ The MongoDB image can be run with mounted volumes to enable persistent storage f
 
 * *_/var/lib/mongodb_* - This is the database directory where
 MongoDB stores database files.
+
+=== Using the MongoDB replication
+
+IMPORTANT: The replication support provided by the MongoDB image is experimental and
+should not be used in production.
+
+This section describes how to start a cluster of MongoDB servers that implements
+master-slave replication and automated failover. This is the MongoDB’s
+recommended replication strategy.
+
+To start with MongoDB replication, you first have to define a DeploymentConfig
```

Comment:
you can break this whole thing down into subsections
(creating the service, creating the primary pod, creating the cluster)